tests/functional/models/awsfooservice/2020-08-20/service-2.json

```diff
@@ -0,0 +1,121 @@
+{
+            "version": "2.0",
+            "metadata": {
+                "apiVersion": "2018-08-31",
+                "endpointPrefix": "fooendpoint",
+                "jsonVersion": "1.1",
+                "protocol": "json",
+                "serviceAbbreviation": "FooService",
+                "serviceId": "FooService",
+                "serviceFullName": "AwsFooService",
+                "signatureVersion": "v4",
+                "signingName": "awsfooservice",
+                "targetPrefix": "awsfooservice"
```

Comment:
Let's rename this to test-discovery-endpoint or something similar to better indicate why we need the model.